_partials/vscode_setup.md

```diff
@@ -1,18 +1,12 @@
-### VS Code Extensions
+### <CODE_EDITOR> Extensions
 
-Let's gain time now and add other extensions that will be helpful during your Bootcamp.
-For each of these extensions:
-- On the web page, click on `install`
-- In the browser, accept to use VS Code to install the extension
-- In VS Code, click on `install`
+Let's gain time now and add other extensions that will be helpful during your Bootcamp:
 
-**Extensions**:
-- [Sublime Text Keymap](https://marketplace.visualstudio.com/items?itemName=ms-vscode.sublime-keybindings)
-- [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
-- [Jupyter](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter)
-- [Python Indent](https://marketplace.visualstudio.com/items?itemName=KevinRose.vsc-python-indent)
+```bash
+<CODE_EDITOR_CMD> --install-extension ms-vscode.sublime-keybindings ms-python.python ms-toolsai.jupyter KevinRose.vsc-python-indent MS-vsliveshare.vsliveshare
+```
 
-### VS Code Settings
+### <CODE_EDITOR> Settings
 
 - On macOS, press `Cmd` + `,` on your keyboard (or `Ctrl` + `,` on Windows or Linux) to open the settings
 - In the search bar, type `emmet`
```

Comment:
There is one setting I have to make on my machine to make VS code works with pyenv:

```bash
{
  "python.pythonPath": "~/.pyenv/shims/python",
}
```

In addition, we should also use exactly the same auto-formatter between jupyter and vscode: Therefore if you told them to install the jupyter extention `code_prettify`, you have to specify `yapf` for VS code too. Remember that `code_prettify` == `yapf` == `pep8 enforcement + nicer prettyfication` (by default) 
```bash
{
  "python.formatting.provider": "yapf", 
}
```

Finally, optionally you may want to add this so they don't clutter their left navbar

```bash
{
  "files.exclude": {
      "**/.pytest_cache": true,
      "**/.ipynb_checkpoints": true,
      "**/__pycache__": true,
      "**/.egg-info": true,
    },
}
```